pkg/storaged/config.json.in

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can this go into an override.json that's installed, and loaded as part of the package manifest?